mslib/mswms/mpl_vsec.py

```diff
@@ -59,6 +61,18 @@ def __init__(self, driver=None):
         """
         super(AbstractVerticalSectionStyle, self).__init__(driver=driver)
 
+    def add_colorbar(self, contour, label=None, tick_levels=None, width="3%", height="30%", format=None):
```

Comment:
Isn't this the same as for the horizontal hsecs?

Reply:
The code between adding colourbars between hsecs and vsecs was slightly different. I added the most common one to their respective super class which reduces the need for providing parameters.